app/views/patients/_fulfillment.html.erb

```diff
@@ -36,7 +36,7 @@
           <% end %>
         </div>
         <div class="info-form-right col-sm-6">
-          <%= f.fields_for patient.fulfillment do |pt| %>
+          <%= f.fields_for @patient.fulfillment do |pt| %>
```

Comment:
Th problem here is that `bootstrap_form_for` doesn't support nested models. Upon further research I've found `bootstrap_nested_form_for` which I suppose we could use instead of the workaround I suggested.

Reply:
Update: `bootstrap_nested_form_for` would require changing the bootstrap version, so I propose we use this workaround for now.

Reply:
Update2: I also looked at the javascript local variables, etc, and it appears we only pass patient, not patient.fulfillment, at least in terms of what we pass explicitly. If we pass patient.fulfillment it may be possible to do this with only the local variable.

Reply:
The plot thickens: it looks like this only fixes what broke in march when fulfillment was made polymorphic, so I'll have to do even more digging into what the problem was before.

Reply:
I think you're right that we don't need to implement some nested forms stuff.

I think what would help me out here is exact steps to recreate the problem -- like, when I do these X things in order in the app using the local variable (no `@`), this doesn't happen and here's how to verify in the rails console, but when I do this with the instance (`@`) this happens, and here's what it looks like in db.

If we need to expedite the model changes to fix #1401  and #1402 then I think we should pull this out into a separate PR, so we don't wind up going down a hole on this.

Reply:
After more testing, I'm not sure why I thought the database wasn't updating, but it is. Weird. I'm backing out the @ changes...

Reply:
I think what led me down this path was that check date/check number weren't saved in production, but it looks like they were just never entered. Lesson learned. Closing this in deference to #1409